app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/DashboardController.kt

```diff
@@ -408,25 +411,29 @@ class DashboardController(
                 .height(16)
                 .addTo(modelList)
 
-            ButtonType2Model_(onCheckSymptomsAgainClick)
+            ButtonType2Model_ { onCheckSymptomsAgainClick(false) }
                 .id("own_health_status_check_symptoms_button")
                 .text(context.string(R.string.self_testing_symptoms_secondary_button))
                 .enabled(exposureNotificationPhase.isReportingEnabled())
-                .onDisabledClick { onFeelingClick(true) }
+                .onDisabledClick { onCheckSymptomsAgainClick(true) }
                 .addTo(modelList)
         }
 
-        if (ownHealthStatus is HealthStatusData.SicknessCertificate) {
+        val isRedRevokingEnabled = dateOfFirstMedicalConfirmation
+            ?.isAfter(ZonedDateTime.now().minusHours(Constants.Behavior.MEDICAL_CONFIRMATION_REVOKING_POSSIBLE_DURATION.toHours()))
```

Comment:
you already called `Duration.ofHours(48)` in the constants definition, is it needed to call `toHours()` again ?

Reply:
You're right, Duration is able to pass directly. 👍 Updated